### PR TITLE
Add-release-notes

### PR DESCRIPTION
**Pull request by AgustinRamiroDiaz**
Info gathered from https://github.com/okteto/app/releases/tag/chart%2F1.2.0-rc.1

### CODE REVIEW

File: src/content/self-hosted/install/releases.mdx [hunk 1]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+- Add permanently user deletion command
+- Allow to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
+- Update okteto cli version to 2.9.0
+
+### Bug Fix
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
```
Comment (maroshii) [inline src/content/self-hosted/install/releases.mdx:25]: We should add a link to this

File: src/content/self-hosted/install/releases.mdx [hunk 2]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+
+- Support for self-signed and private CA with Docker
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
+- Allow to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
+- Update okteto cli version to 2.9.0
```
Comment (maroshii) [inline src/content/self-hosted/install/releases.mdx:20]: Add link to CLI release page

File: src/content/self-hosted/install/releases.mdx [hunk 3]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+### New Feature
+
+- Support for self-signed and private CA with Docker
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
+- Allow to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
```
Comment (maroshii) [inline src/content/self-hosted/install/releases.mdx:19]: The change is not about the helm values updates but more about the functionality. I see we now have `podSecurityPolicy.enabled=true` by default (with AppArmour enabled). Did this functionality change? If so we probably need to be a little more descriptive

File: src/content/self-hosted/install/releases.mdx [hunk 4]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
+- Update text in error when using token-based auth
+- Persist selfsigned wildcard certificate
+- Update daemon image
+- Allow pod to have its own image pull secret
+- Fix "Uncaught ReferenceError: $RefreshReg$ is not defined"
```
Comment (maroshii) [inline src/content/self-hosted/install/releases.mdx:32]: This is vert cryptic. Where did it happen? I would just delete from the notes

File: src/content/self-hosted/install/releases.mdx [hunk 5]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+### Bug Fix
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
+- Update text in error when using token-based auth
+- Persist selfsigned wildcard certificate
+- Update daemon image
+- Allow pod to have its own image pull secret
```
Comment (maroshii) [inline src/content/self-hosted/install/releases.mdx:31]:
```suggestion
- Allow pods to have their own image pull secret
```

File: src/content/self-hosted/install/releases.mdx [hunk 6]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+- Fix "Uncaught ReferenceError: $RefreshReg$ is not defined"
+- Add registry hostname to installer /etc/hosts
+- Fix user deletion when there is no owner among the namespace members
+- Fix pending to sleep label
```
Comment (maroshii) [inline src/content/self-hosted/install/releases.mdx:35]: We can probably get rid of this. If not, we should describe a little more what the fix is

File: src/content/self-hosted/install/releases.mdx [hunk 7]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.2.0
+
+### New Feature
+
+- Support for self-signed and private CA with Docker
```
Comment (pchico83) [inline src/content/self-hosted/install/releases.mdx:12]:
```suggestion
- Support for self-signed certificates and private CA with Docker
```

File: src/content/self-hosted/install/releases.mdx [hunk 8]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+- Buildkit port is now configurable (default 443)
+- Add option for sharing namespace with users within organization
+- UI pagination support for GitHub branches list
+- Add permanently user deletion command
```
Comment (pchico83) [inline src/content/self-hosted/install/releases.mdx:16]: is this a Cloud feature? or a self-hosted one?

**Review by pchico83 — APPROVED** [review]
@AgustinRamiroDiaz can we add links to the docs for those features that have docs?

File: src/content/self-hosted/install/releases.mdx (hunk 7 above)
Comment (rberrelleza) [reply src/content/self-hosted/install/releases.mdx:12]: What does the "with docker" part refers to?

File: src/content/self-hosted/install/releases.mdx [hunk 9]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
+- Update text in error when using token-based auth
+- Persist selfsigned wildcard certificate
+- Update daemon image
+- Allow pod to have its own image pull secret
+- Fix "Uncaught ReferenceError: $RefreshReg$ is not defined"
+- Add registry hostname to installer /etc/hosts
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:33]: Let's rephrase this to be about the benefit or use case this enables

File: src/content/self-hosted/install/releases.mdx [hunk 10]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+- Fix user deletion when there is no owner among the namespace members
+- Fix pending to sleep label
+- Fix issues showing virtual service endpoints
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:36]: Rephrase to be about the user benefit ("virtual services are now correctly displayed in the dashboard " or something like that)

File: src/content/self-hosted/install/releases.mdx [hunk 11]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
+- Update text in error when using token-based auth
+- Persist selfsigned wildcard certificate
+- Update daemon image
+- Allow pod to have its own image pull secret
+- Fix "Uncaught ReferenceError: $RefreshReg$ is not defined"
+- Add registry hostname to installer /etc/hosts
+- Fix user deletion when there is no owner among the namespace members
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:34]: I think the fix is that the admin will be the owner? We should mention that.

File: src/content/self-hosted/install/releases.mdx [hunk 12]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+
+### Bug Fix
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
+- Update text in error when using token-based auth
+- Persist selfsigned wildcard certificate
+- Update daemon image
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:30]: Update to what? If it's the internal update don't mention it.

File: src/content/self-hosted/install/releases.mdx [hunk 13]
```diff
@@ -5,6 +5,36 @@ sidebar_label: Release Notes
...
+- Populate oauth errors to the user
+- Added new helm values to set PSP and AppArmor
+- Update okteto cli version to 2.9.0
+
+### Bug Fix
+
+- Don't show warning on k8s 1.23.x
+- Frontend nginx CVE-2022-40303/CVE-2022-40304
+- Fix loader icon regression
+- Fix in license display
```
Comment (rberrelleza) [inline src/content/self-hosted/install/releases.mdx:27]: What was fixed?

File: src/content/self-hosted/install/releases.mdx (hunk 12 above)
Comment (maroshii) [reply src/content/self-hosted/install/releases.mdx:30]: We can delete this. The update is the `Add registry hostname to installer /etc/hosts` mentioned above

File: src/content/self-hosted/install/releases.mdx (hunk 8 above)
Comment (AgustinRamiroDiaz) [reply src/content/self-hosted/install/releases.mdx:16]: I think we did it for self-hosted. @AdrianPedriza can you confirm?

File: src/content/self-hosted/install/releases.mdx (hunk 1 above)
Comment (AgustinRamiroDiaz) [reply src/content/self-hosted/install/releases.mdx:25]: @maroshii You mean a link to the CVE? like this one https://ubuntu.com/security/CVE-2022-40303?

File: src/content/self-hosted/install/releases.mdx (hunk 8 above)
Comment (AdrianPedriza) [reply src/content/self-hosted/install/releases.mdx:16]: it is a Cloud feature

File: src/content/self-hosted/install/releases.mdx (hunk 13 above)
Comment (AgustinRamiroDiaz) [reply src/content/self-hosted/install/releases.mdx:26]: The seats https://github.com/okteto/app/issues/4799

File: src/content/self-hosted/install/releases.mdx (hunk 9 above)
Comment (AgustinRamiroDiaz) [reply src/content/self-hosted/install/releases.mdx:30]: I've rephrased it. @jpf-okteto can you confirm?

File: src/content/self-hosted/install/releases.mdx (hunk 11 above)
Comment (AgustinRamiroDiaz) [reply src/content/self-hosted/install/releases.mdx:31]: reading the pr changes (https://github.com/okteto/app/pull/4905) it seems like we are just bypassing the problem for now, wo the admin is not being assigned here

File: src/content/self-hosted/install/releases.mdx (hunk 3 above)
Comment (AgustinRamiroDiaz) [reply src/content/self-hosted/install/releases.mdx]: I've added a link to the docs. I've changed the description to express that this is now configurable

File: src/content/self-hosted/install/releases.mdx (hunk 1 above)
Comment (maroshii) [reply src/content/self-hosted/install/releases.mdx:23]: yes

**Comment by AdrianPedriza** [comment]
I wonder if we should mention items like: 

- UI pagination support for GitHub branches list
- Fix loader icon regression
- Fix user deletion when there is no owner among the namespace members

would be transparent to the user, wouldn't it be internal? On the other hand, maybe it would be a good idea to have the release notes done jointly between the developer and someone closer to the customer with a clearer view of what should be exposed to the customer...? cc @rberrelleza @pchico83 @omnomagonz

File: src/content/self-hosted/install/releases.mdx (hunk 13 above)
Comment (AdrianPedriza) [reply src/content/self-hosted/install/releases.mdx]: Maybe adding the `Fix` word is redundant due to we are in the `BugFix` section. In this case mentioned above, would it not be better: `Display license related information properly` ?

File: src/content/self-hosted/install/releases.mdx [hunk 14]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Save costs by allowing to scale down to zero nodes if no user pods are running
+- Populate oauth errors to the user
+- Added [configuration to set PSP and AppArmor](/docs/self-hosted/administration/configuration/#podSecurityPolicy)
+- Update okteto cli version to [2.9.0](https://github.com/okteto/okteto/releases/tag/2.9.0)
+
+### Bug Fix
+
+- Don't show warning when installing okteto on k8s 1.23.x
+- Frontend nginx [CVE-2022-40303](https://access.redhat.com/security/cve/CVE-2022-40303)/[CVE-2022-40304](https://access.redhat.com/security/cve/CVE-2022-40304)
+- Fix loader icon regression
```
Comment (AdrianPedriza) [inline src/content/self-hosted/install/releases.mdx:24]: I think `Fix` here is redundant

File: src/content/self-hosted/install/releases.mdx [hunk 15]
```diff
@@ -5,6 +5,31 @@ sidebar_label: Release Notes
...
+- Display license related information properly
+- Update text in error when using token-based auth
+- Persist selfsigned wildcard certificate
+- Allow pods to have their own image pull secret
+- Added `registry's hostname` to `installer pods` (`/etc/hosts`) pointing to the internal IP, to ensure an internal connection to the component.
+- Fix user deletion when there is no owner among the namespace members
```
Comment (AdrianPedriza) [inline src/content/self-hosted/install/releases.mdx:30]: I think `Fix` here is redundant

**Review by AdrianPedriza — APPROVED** [review]

**Pull request merged**